wordpress-mobile/WordPress-Android
# Issue/add share site

### Fix
Add an action to share sites in the Reader for blogs and feeds.  The share site behavior mimics the share post behavior.  See https://github.com/wordpress-mobile/WordPress-iOS/pull/10899 for details about the iOS counterpart changes.  See the screenshots below for illustration.

![share_site](https://user-images.githubusercontent.com/3827611/55594986-5609af00-5731-11e9-88ca-abfede432509.png)

### Test
1. Go to ***Reader*** tab.
2. Tap site button in header of post.
3. Notice ***Share*** action in top app bar.
4. Tap ***Share*** action in top app bar.
5. Notice share action sheet is shown.
6. Tap email app in list.
7. Notice blog/feed name in email subject and blog/feed URL in email content.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
The `RELEASE-NOTES.txt` document was updated in 9c87a14 with the following:
> Added an action to share sites in the Reader
